pkg/client/grpc.go

```diff
@@ -176,6 +176,41 @@ func (client *grpcClient) Read(c *cli.Context, rack, board, device string) (out
 	return out, nil
 }
 
+// ReadCached issues a "readcached" request to a plugin via the gRPC API.
+// FIXME - Small note on the use of params and consistency:
+// Here we specify the params of the starting and ending bound to be string.
+// However, in the client.go, we use a struct parameter. Need to think more
+// about this and chose to go with either one of these for consistency?
+func (client *grpcClient) ReadCached(c *cli.Context, start, end string) (out []*synse.DeviceReading, err error) {
```

Comment:
+1

I think it would make sense to pass along the ReadCachedParams struct, since the start and end are defined therein as strings. Not a huge deal either way, but you're right that it would make its usage more consistent

Reply:
For this one, I am trying to see if refactoring all functions to use a struct params is more worthwhile than justing strings as normal.

Also, there are some stuff here and there related to scheme and formatter and the way we do thing is somewhat inconsistent. Will try to look into that in a different pr.